config/main.py

```diff
@@ -903,6 +1107,8 @@ def remove(ctx, ip_addr):
 
     if interface_name.startswith("Ethernet"):
         config_db.set_entry("INTERFACE", (interface_name, ip_addr), None)
+    elif interface_name.startswith("eth0"):
```

Comment:
@pavel-shirshov 
pavel-shirshov 11 days ago Contributor 
interface_name == 

Will address this.

Reply:
@pavel-shirshov : Addressed your comment. Changed "startswith eth0", to "exact match for eth0"